app/lib/postcodeLinks.js

```diff
@@ -31,6 +31,14 @@ function addUrl(postcodeLocationDetails, inputList) {
   });
 }
 
+function addExternalUrl(postcodeLocationDetails, location) {
```

Comment:
Rather than adding this function to this module I suggest a new module is created to keep a single responsibility.
Also, the function isn't adding a URL, it is merely returning one. Please rename to something like `getChoicesResultsUrlToOnlineTests`.

Reply:
Given they both generate links based on lat lon from a postcode it felt right adding them in here.  Will change the name, though.

Reply:
Typically, modules are named by what they do rather than what they use to do what they do.
Also, it is common to use small, well defined modules, each doing one thing rather than bundling functions that may only be tangentially linked with one another into a module (obviously this isn't always the case and modules like the incredibly generically named `utils` are an exception to this rule but perhaps not a good thing!).
The name of this module is not going to help people in the future looking for these functions. If the functions are to exist within the same module, it should be renamed to indicate what functionality it provides. Something like `externalUrlGenerator` would do it.